routes/index.js

```diff
@@ -5,9 +5,20 @@ const db = require('../db/models');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'GoodLaughs' });
+  let username = [];
```

Comment:
I'm curious why this is an array. I'm assuming you have some sort of possibility in your template of rendering out multiple usernames.